comictaggerlib/cli.py

```diff
@@ -82,6 +82,8 @@ def actual_metadata_save(self, ca: ComicArchive, md: GenericMetadata) -> bool:
                 logger.info("dry-run option was set, so nothing was written, but here is the final set of tags:")
                 print("dry-run option was set, so nothing was written, but here is the final set of tags:")
                 print(f"{md}")
+
+        print(f"Metadata provided by {self.current_talker().name} {self.current_talker().website}")
```

Comment:
This is not a good place for this. This will show for every comic that is saved. This can be listed once when the CLI is started or once when it is ended.